gnatsd_streaming/ci/gnatsd_streaming.rake

```diff
@@ -0,0 +1,128 @@
+require 'ci/common'
+
+def gnatsd_streaming_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def gnatsd_streaming_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/gnatsd_streaming_#{gnatsd_streaming_version}"
+end
+
+def gnatsd_streaming_temp_dir
+  ENV['VOLATILE_DIR'] + '/gnatsd_streaming'
+end
+
+module GnatsdDocker
+  def volumes
+    "--volume=#{gnatsd_streaming_temp_dir}/data:/usr/share/nats/data"
+  end
+
+  def publish
+    '--publish=4222:4222 --publish=8222:8222'
+  end
+
+  def store_mount
+    '-store file -dir /usr/share/nats/data'
+  end
+
+  def primary_routes
+    "-cluster nats://#{container_name}-primary:5222 -routes nat://#{container_name}-secondary:5222 -ft_group failover"
+  end
+
+  def secondary_routes
+    "-cluster nats://#{container_name}-secondary:5222 -routes nat://#{container_name}-primary:5222 -ft_group failover"
+  end
+
+  def primary
+    "nats-streaming:0.7.0 #{store_mount} #{primary_routes} -m 8222"
```

Comment:
There have been few releases since PR open... maybe bump already to `0.9.2` here already? It would not affect the test either way though